caffe2/opt/onnxifi_op.h

```diff
@@ -261,7 +261,6 @@ class OnnxifiOp final : public Operator<Context> {
               weight_descs.size(),
               weight_descs.data(),
               &graph,
-              static_cast<uint32_t>(max_seq_size_),
```

Comment:
We use a variant of onnxifi in our usecase:
https://github.com/houseroad/foxi/blob/master/foxi/onnxifi.h#L1767

I believe the pytorch repro is built with the above repo. And we cannot take this out.

Reply:
@yinghai when you use a own variant to build pytorch, why is the then 'official' pytorch https://github.com/onnx/onnx/tree/a82c6a7010e2e332d8f74ad5b0c726fd47c85376
referenced in third_party?
Do you or anyone else also know if there is an option to block the build of onnxifi?

Reply:
foxi is used in third_party of pytorch https://github.com/pytorch/pytorch/tree/master/third_party. I guess there might be some mixed up in the headers? @malfet, has anything changed in the build system recently to trigger this issue?

Reply:
This issue is triggered by 'our' build system see https://build.opensuse.org/package/show/science:machinelearning/python-torch where we do not include foxi.
Still this is a ugly header header mess up, as you have two different versions of onnxifi.h in third_party, the foxi one and the onnx one.